Jenkinsfile

```diff
@@ -106,6 +102,20 @@ pipeline {
             }
         }
 
+        stage('DDF Core Tests Only Build') {
```

Comment:
Do you know how long it is taking the core tests to run? If it is less than an hour I like setting the timeout to 1 hour to keep a standard of not making exceed an hour.

Reply:
Core tests run under 10-15 mins, I set timeout to one hour for ITests and Core tests